library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -206,104 +259,298 @@ TensileStatus tensile_Cijk_Alik_Bjlk_B<TensileHalf, TensileHalf, float>(TENSILE_
 }
-//---typename_input=int8----typename_output=int------typename_compute=int--------------------------
+
+//----- typename_input = int8 ---- typename_output = int ------ typename_compute = int ------------------
 template <>
-TensileStatus tensile_Cijk_Ailk_Bljk_B<TensileInt8x4, TensileInt32, TensileInt32>(
+inline TensileStatus tensile_Cijk_Ailk_Bljk_B<TensileInt8x4, TensileInt32, TensileInt32>(
     TENSILE_IN_ARGS(TensileInt8x4, TensileInt32, TensileInt32))
 {
     return tensile_Cijk_Ailk_Bljk_4xi8BH(TENSILE_OUT_ARGS);
 }
 template <>
-TensileStatus tensile_Cijk_Ailk_Bjlk_B<TensileInt8x4, TensileInt32, TensileInt32>(
+inline TensileStatus tensile_Cijk_Ailk_Bjlk_B<TensileInt8x4, TensileInt32, TensileInt32>(
     TENSILE_IN_ARGS(TensileInt8x4, TensileInt32, TensileInt32))
 {
     return tensile_Cijk_Ailk_Bjlk_4xi8BH(TENSILE_OUT_ARGS);
 }
 template <>
-TensileStatus tensile_Cijk_Alik_Bljk_B<TensileInt8x4, TensileInt32, TensileInt32>(
+inline TensileStatus tensile_Cijk_Alik_Bljk_B<TensileInt8x4, TensileInt32, TensileInt32>(
     TENSILE_IN_ARGS(TensileInt8x4, TensileInt32, TensileInt32))
 {
     return tensile_Cijk_Alik_Bljk_4xi8BH(TENSILE_OUT_ARGS);
 }
 template <>
-TensileStatus tensile_Cijk_Alik_Bjlk_B<TensileInt8x4, TensileInt32, TensileInt32>(
+inline TensileStatus tensile_Cijk_Alik_Bjlk_B<TensileInt8x4, TensileInt32, TensileInt32>(
     TENSILE_IN_ARGS(TensileInt8x4, TensileInt32, TensileInt32))
 {
     return tensile_Cijk_Alik_Bjlk_4xi8BH(TENSILE_OUT_ARGS);
 }
+
+//----- typename_data=rocblas_float_complex ---------- typename_compute = rocblas_float_complex --------------------------
+#define TENSILE_COPMLEX_OUT_ARGS(Ti, To, Tc)                                        \
+    (To*)dataD, (const To*)dataC, (const Ti*)dataA, (const Ti*)dataB,               \
+        *((Tc*)&alpha), *((Tc*)&beta),                                              \
+        strideD1J, strideD2K, strideC1J, strideC2K,                                 \
+        strideA1L, strideA2K, strideB1J, strideB2K,                                 \
+        sizeI, sizeJ, sizeK, sizeL, stream, 0, nullptr, nullptr
+
+static_assert(std::is_standard_layout<TensileComplexFloat>{},
+          "rocblas_float_complex is not a standard layout type, and thus is "
+          "incompatible with C.");
+
+static_assert(std::is_trivial<TensileComplexFloat>{},
+          "rocblas_float_complex is not a trivial type, and thus is "
+          "incompatible with C.");
+
+static_assert(sizeof(rocblas_float_complex) == sizeof(TensileComplexFloat),
+          "internal rocblas_float_complex does not match public rocblas_float_complex");
+template <>
+inline TensileStatus tensile_Cijk_Ailk_Bljk_B<rocblas_float_complex,rocblas_float_complex,rocblas_float_complex>(
+    TENSILE_IN_ARGS(rocblas_float_complex, rocblas_float_complex, rocblas_float_complex))
+{
+    return tensile_Cijk_Ailk_Bljk_CB(TENSILE_COPMLEX_OUT_ARGS(TensileComplexFloat, TensileComplexFloat, TensileComplexFloat));
```

Comment:
Same typo throughout file -- maybe do global search and replacce

Reply:
understand. update it in commit "update complex change 2".